.github/workflows/ci.yml

```diff
@@ -9,6 +9,7 @@ env:
   ELIXIR_VERSION: 1.13
   OTP_VERSION: 24.2
   MIX_ENV: test
+  TOKENIZERS_BUILD: "true"
```

Comment:
Do we use this anywhere?

Reply:
In the "native" module. We want to always force the build in the CI. See: https://github.com/elixir-nx/tokenizers/blob/122239907580036a5c26cfe35f1eae2355f3d4d0/lib/tokenizers/native.ex#L11